src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -137,7 +124,7 @@ export const ProjectListItemCard: FC<ProjectListItemCardProps> = ({
               <Box component="span" m={1} display="inline" color="text.primary">
                 ESAD:
               </Box>
-              {esadDate}
+              {props.estimatedSubmission?.value}
```

Comment:
We should hide the label if there is no value. Also let's not use `Box` here for performance reasons.